pkg/virt-config/config_test.go

```diff
@@ -256,4 +256,15 @@ var _ = Describe("ConfigMap", func() {
 		clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{})
 		Expect(clusterConfig.GetMachineType()).To(testutils.SatisfyAnyRegexp(clusterConfig.GetEmulatedMachines()))
 	})
+
+	table.DescribeTable("SMBIOS values from kubevirt-config", func(value string, result virtconfig.SmbiosConfig) {
+		clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+			Data: map[string]string{virtconfig.SmbiosConfigKey: value},
+		})
+		smbios := clusterConfig.GetSMBIOS()
+		Expect(smbios).To(Equal(result))
+	},
+		table.Entry("When no values are set, should be nil", `{"smbios":{}`, virtconfig.SmbiosConfig{}),
+		table.Entry("when values set, should equal to result", `{"smbios":{"Family":"KubeVirt","Product":"KubeVirt", "Manufacturer":"None"}}`, virtconfig.SmbiosConfig{Family: "KubeVirt", Product: "KubeVirt", Manufacturer: "None"}),
```

Comment:
the `smbios` key in your json is not necessary as you already have a key when you are setting the value `Data: map[string]string{virtconfig.SmbiosConfigKey: value},`

Reply:
corrected